internal/dataplane/parser/translate_httproute.go

```diff
@@ -249,12 +244,12 @@ func generateKongRouteFromTranslation(
 func generateKongRouteFromHTTPRouteMatches(
```

Comment:
If the return type has changed shouldn't this also reflect that in the name?

```suggestion
func generateKongRoutesFromHTTPRouteMatches(
```